Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Minseong Kang on 2023/05/30.
+//
+
+
+protocol CalculateItem {
+	
+}
```

Comment:
빈 프로토콜을 선언할 때는 대괄호 사이를 한칸 띄어쓰는게 일반적인 방법입니다. 정답은 아니니 참고하세용~
```suggestion
protocol CalculateItem { }
```

Reply:
네!! 꼭 참고하겠습니다 🙋‍♂️